libraries/node/include/node/node.hpp

```diff
@@ -69,7 +68,6 @@ class FullNode : public std::enable_shared_from_this<FullNode> {
   std::shared_ptr<DagBlockManager> dag_blk_mgr_;
   std::shared_ptr<TransactionManager> trx_mgr_;
   std::shared_ptr<Network> network_;
-  std::shared_ptr<TransactionOrderManager> trx_order_mgr_;
```

Comment:
TransactionOrderManager is now used only in single unit test -> DagBlockTest.overlap. I would move this class declaration directly into the test or move files somewhere next to the test so it is not part of folder with source code that is actually used in node...

Reply:
As you wrote in https://github.com/Taraxa-project/taraxa-node/pull/928#discussion_r718244810 TransactionOrderManager wasn’t used anywhere, but that test. So I removed it and that related test. I don't think we need to test unused functionality